reference/twig_reference.rst

```diff
@@ -324,6 +324,10 @@ Returns the relative URL (without the scheme and host) for the given route.
 If ``relative`` is enabled, it'll create a path relative to the current
 path. More information in :ref:`templating-pages`.
 
+.. note::
+
+    More information about routing in :ref:`routing`.
```

Comment:
I would put this in an `seealso` block. And then I would also reword this a bit (please also not that you have to use the `doc` role):

```rst
.. seealso::

    Read :doc:`/routing` to learn more about the Routing component.
```